### ISSUE TITLE
latex: Do not use Babel.get_language().

### ISSUE THREAD

**Issue by tk0miya** [sphinx-doc/sphinx#4877]
### Problem
- The method was marked as obsoleted. And it was kept only for Sphinx.

```
    def get_language(self):
        # Obsolete, kept for backwards compatibility with Sphinx
        return self.language
```

- I checked docutils-0.11 which is oldest one supported by current release. It contains above comment.

#### Expected results
- Do not use the method.

**Comment by jfbu** [comment]
+1

**Comment by tk0miya** [comment]
Fixed by #4940
